bitcoind-rpc/src/Bitcoin/Core/RPC/Blockchain.hs

```diff
@@ -32,37 +34,39 @@ module Bitcoin.Core.RPC.Blockchain (
 import Bitcoin.CompactFilter (BlockFilter, BlockFilterHeader)
 import Data.Aeson (
     FromJSON (..),
+    Value (String),
     withObject,
     withText,
     (.:),
     (.:?),
  )
 import Data.Aeson.Types (Parser)
+import Data.Aeson.Utils (
+    HexEncoded (unHexEncoded),
+    decodeFromHex,
+    toSatoshis,
+    utcTime,
+ )
 import Data.Int (Int64)
+import Data.Maybe (fromMaybe)
 import Data.Proxy (Proxy (..))
 import Data.Scientific (Scientific)
 import Data.Serialize (Serialize)
 import Data.Text (Text)
 import Data.Time (NominalDiffTime, UTCTime)
+import Data.Time.Clock.POSIX (utcTimeToPOSIXSeconds)
 import Data.Word (Word16, Word32, Word64)
-import Haskoin.Block (Block, BlockHash, BlockHeight, hexToBlockHash)
+import Haskoin.Block (Block (..), BlockHash, BlockHeight, hexToBlockHash)
+import qualified Haskoin.Block as Haskoin
```

Comment:
Changing this to `import qualified Haskoin as H` would make the import consistent with the rest of the codebase.  I think `Haskoin` exports the `Block` accessors.

Reply:
Ok I checked and yes it does export the `Block` accessors, I've changed it now, good suggestion 👍🏼.